louis/db/api/__init__.py

```diff
@@ -1,5 +1,33 @@
-from louis.models import openai
+#from louis.models import openai
+import louis.models.openai as openai
+import os
+import json
+import sys
+import louis.db as db
+import dotenv
+# This is used to load the .env file
+dotenv.load_dotenv()
 
+FINESSE_WEIGHTS = os.environ.get("FINESSE_WEIGHTS") \
+    or db.raise_db_error("FINESSE_WEIGHTS is not set")
+
+# should be something like that : '{
+#     "recency": 1,
+#     "traffic": 1,
+#     "current": 0.5,
+#     "typicality": 0.2,
+#     "similarity": 1
+# }'
```

Comment:
I suggest removing this for a cleaner code.

Reply:
done